### PR TITLE
Add explanation of sync conflicts, and sync not running in background

### PR DESCRIPTION
**Pull request by claremacrae**
This is to try and capture some useful info from the obsidian-sync Discord channel this morning, including the info:

https://discord.com/channels/686053708261228577/900870622631063552/907925338674311200

> They are merged with Google's diff match patch algorithm, which normally does a good job but I've seen it sometimes just adds one version to the other

I started with all the new text being in the FAQ section, but it got a bit long, so I moved it to the main body.

### CODE REVIEW

**Comment by claremacrae** [comment]
I hummed and hahed about whether to include the bullet list with the example scenario...

I included it, but would be totally OK with it being deleted if it was felt that text was too wordy...

File: en/Licenses & add-on services/Obsidian Sync.md [hunk 1]
```diff
@@ -60,6 +60,22 @@ You can right click a file in the file explorer pane to see its version history.
...
+- you edit a file on a mobile device without an internet connection.
+- then you make different edits to the same file on a desktop machine, and those changes get uploaded by Obsidian Sync
+- later you return to the vault on the mobile device and have an internet connection
+- now Obsidian downloads changes and finds this file has been changed in different places, and has to resolve the conflicts.
+
+When Sync downloads a new version of a file, and finds that there are conflicts with the local copy, the changes are merged with Google's diff-match-patch algorithm.
+
+This normally works well, although sometimes it does sometime just add one version of the file to the other.
+
+Note that unless you search the sync log for `Merging conflicted file`, there is no visible notification if a conflict has happened, and how it was resolved.
```
Comment (ericaxu) [inline en/Licenses & add-on services/Obsidian Sync.md:77]: Is this true? I think there's a pop-up saying "{filename} has been edited externally, merging conflicts automatically." when it happens.

Comment (kometenstaub) [reply en/Licenses & add-on services/Obsidian Sync.md:77]: That happens when there is an external merge conflict, so if the file was changed from another program on the same computer. When sync merges the local and remote file, then it will only show up in the sync log. In addition, the merging behaviour has been altered in 0.13.24 so as to keep the latest version.

**Comment by ericaxu** [comment]
Unfortunately it's too much work to merge it into the existing docs now that it has been rewritten (see https://help.obsidian.md/Obsidian+Sync/Introduction+to+Obsidian+Sync). Sorry for not merging it sooner.

I also spent half an hour trying to find a good place for it, such as in "Limitation" and "Troubleshooting", but they all feel wrong. If you want do another PR based on the latest docs.

While I understand you probably wrote it after you had a conflict, would appreciate it if you can try to keep the tone neutral and keep the reader's current perspective in mind.

For example, I rewrote your original

> Note that unless you search the sync log for `Merging conflicted file`, there is no visible notification if a conflict has happened, and how it was resolved.

into:

> Tip: To find when conflicts have happened, you can search for "Merging conflicted file" in **Settings** → **Sync** → **Sync activity** → **View**.

Thanks!
